### PR TITLE
Speed up EE tests by not destroying tables

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Destroying and recreating clickhouse tables takes about 1.5s per test. Instead, let's just create a new team and filter by that team for everything. Might also open up parallelisation.

- Don't destroy and recreate clickhouse tables between tests. This took about 1.5s per test. Instead filter all tests by team_id
- Also stopped running clickhouse migrations before boot, instead manually create every table. Running migrations takes ~40 seconds, creating all tables like this takes ~4 seconds. Annoyingly we will have to remember to add a table there if we add new tables, but probably worth the tradeoff
- Fix a few very slow tests by reducing the amount of data we create

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: ee/clickhouse/models/test/test_action.py [hunk 1]
```diff
@@ -50,6 +50,9 @@ def _create_person(**kwargs) -> Person:
     return Person(id=person.uuid)
 
 
+EVENT_UUID_QUERY = "SELECT uuid FROM events WHERE {} AND team_id = %(team_id)s"
```
Comment (EDsCODE) [inline ee/clickhouse/models/test/test_action.py:53]: Did clean up here just so it's reused

File: ee/conftest.py [hunk 2]
```diff
@@ -32,9 +71,7 @@ def django_db_setup(django_db_setup, django_db_keepdb):
     if not django_db_keepdb or not database.db_exists:
         database.create_database()
 
-    database.migrate("ee.clickhouse.migrations")
-    # Make DELETE / UPDATE synchronous to avoid flaky tests
-    sync_execute("SET mutations_sync = 1")
```
Comment (EDsCODE) [inline ee/conftest.py:37]: Not needed anymore. the delete_person test was solved by setting the configuration for clickhouse overall [here](https://github.com/PostHog/posthog/pull/5732)

**Review by EDsCODE — APPROVED** [review]
lgtm. Thanks for the fixes. Looks like tests are running 50-70% faster

**Comment by timgl** [comment]
Thanks for looking into that error! It had me stumped.

**Pull request merged**